src/wp-admin/includes/class-wp-filesystem-ftpext.php

```diff
@@ -602,7 +602,20 @@ public function rmdir( $path, $recursive = false ) {
 
 	/**
 	 * @param string $line
-	 * @return array
+	 * @return array {
+	 *     Array of file information.
+	 *
+	 *     @type string       $name        Name of the file or directory.
+	 *     @type string       $perms       *nix representation of permissions.
+	 *     @type string       $permsn      Octal representation of permissions.
+	 *     @type string|false $owner       Owner name or ID.
+	 *     @type string|false $size        Size of file in bytes as a string.
+	 *     @type string|false $lastmodunix Last modified unix timestamp as a string.
+	 *     @type string|false $lastmod     Last modified month (3 letter) and day (without leading 0).
+	 *     @type string|false $time        Last modified time.
+	 *     @type string       $type        Type of resource. 'f' for file, 'd' for directory, 'l' for link.
+	 *     @type array        $files       If a directory and `$recursive` is true, contains another array of files.
+	 * }
```

Comment:
This is for `parselisting()`. Is it intentional?

Reply:
Yep 👍